src/Form/Type/BaseStatusType.php

```diff
@@ -93,6 +103,12 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $choices = call_user_func([$this->class, $this->getter]);
 
+        // choice_as_value options is not needed in SF 3.0+
+        if (method_exists(FormTypeInterface::class, 'setDefaultOptions')) {
```

Comment:
I think @EmmanuelVella has a better way for this, `$resolver->defined('choices_as_value') ` iirc